includes/abstracts/llms.abstract.notification.view.quiz.completion.php

```diff
@@ -133,7 +134,7 @@ protected function set_merge_data( $code ) {
 				break;
 
 			case '{{GRADE}}':
-				$code = round( $attempt->get( 'grade' ), 2 ) . '%';
+				$code = round( $attempt->get( 'grade' ) ?? 0, 2 ) . '%';
```

Comment:
Do we want to show this as 0% or blank? 0% implies they took it but got zero questions correct.